page-id.ts

```diff
@@ -103,8 +103,8 @@ export enum PageId {
   // thirdweb.com/chainlist
   ChainsLanding = "chains-landing",
 
-  // thirdweb.com/hackathon/solanathon
-  // SolanaHackathonLanding = "solanathon",
+  // thirdweb.com/hackathon
+  HackathonLanding = "hackathon",
```

Comment:
this looks wrong? should be something related to BaseConsumerCrypto not generic hackathon